### PR TITLE
Update intro

### PR DESCRIPTION
**Pull request by heitortsergent**
- Add link to Google's style guide to first page
- Add link to GitHub repo

### CODE REVIEW

File: docs/sources/_index.md [hunk 1]
```diff
@@ -33,6 +33,14 @@ Writers’ Toolkit is for anyone who writes or edits customer-facing technical d
 
 We write our documentation using U.S. English.
 
+Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in the Writers' Toolkit, refer to Google's style guide.
+
+If you're new to the guide, start with the [Get started]({{< relref "./get-started" >}}) section.
+
```
Comment (heitortsergent) [inline docs/sources/_index.md:39]: I think the flow of this introduction section could be improved, but I'm not sure how to go about it.

File: docs/sources/_index.md [hunk 2]
```diff
@@ -33,6 +33,14 @@ Writers’ Toolkit is for anyone who writes or edits customer-facing technical d
 
 We write our documentation using U.S. English.
 
+Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in the Writers' Toolkit, refer to Google's style guide.
```
Comment (jdbaldry) [inline docs/sources/_index.md:36]: nit: I believe it should be just "Writers' Toolkit" without "the".

 ```suggestion
Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in Writers' Toolkit, refer to Google's style guide.
```

File: docs/sources/_index.md [hunk 3]
```diff
@@ -33,6 +33,14 @@ Writers’ Toolkit is for anyone who writes or edits customer-facing technical d
...
+
+If you're new to the guide, start with the [Get started]({{< relref "./get-started" >}}) section.
+
 ## Iteration is our friend
 
 This is just a start. We will continue to share our guidelines and seek feedback and iterate. Our hope is that these pages and our understanding continues and evolves over time. Over time, we will expand Writers’ Toolkit.
+
+## Questions and Feedback
+
+The Writers' Toolkit is open-source and available at [grafana/writers-toolkit](https://github.com/grafana/writers-toolkit). If you have questions, or feedback on how we can improve it, we encourage you to open an issue and help us make this an even better resource.
```
Comment (jdbaldry) [inline docs/sources/_index.md:46]:
```suggestion
Writers' Toolkit is open-source and available at [grafana/writers-toolkit](https://github.com/grafana/writers-toolkit). If you have questions, or feedback on how we can improve it, we encourage you to open an issue and help us make this an even better resource.
```

File: docs/sources/_index.md (hunk 2 above)
Comment (heitortsergent) [reply docs/sources/_index.md]: Good catch, thanks! 🙏

File: docs/sources/_index.md [hunk 4]
```diff
@@ -33,6 +33,14 @@ Writers’ Toolkit is for anyone who writes or edits customer-facing technical d
...
 
+Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in Writers' Toolkit, refer to Google's style guide.
+
+If you're new to the guide, start with the [Get started]({{< relref "./get-started" >}}) section.
+
 ## Iteration is our friend
 
 This is just a start. We will continue to share our guidelines and seek feedback and iterate. Our hope is that these pages and our understanding continues and evolves over time. Over time, we will expand Writers’ Toolkit.
+
+## Questions and Feedback
```
Comment (josmperez) [inline docs/sources/_index.md:44]:
```suggestion
## Questions and feedback
```

Comment (heitortsergent) [reply docs/sources/_index.md]: Thanks @josmperez!

**Review by jdbaldry — APPROVED** [review]
LGTM, feel free to merge :)

**Pull request merged**